ci/k8s/deployment.yml

```diff
@@ -79,12 +79,16 @@ spec:
             port: 8000
           initialDelaySeconds: 30
           timeoutSeconds: 1
+          failureThreshold: 12
+          periodSeconds: 10
```

Comment:
This is the default value

Reply:
Correct, added it for those of us that do not know the defaults by heart